src/content/en/fundamentals/performance/optimizing-javascript/code-splitting/index.md

```diff
@@ -0,0 +1,745 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: TBD
+
+{# wf_updated_on: 2018-07-30 #}
+{# wf_published_on: 2018-06-14 #}
+{# wf_blink_components: Blink>JavaScript #}
+
+# Reduce JavaScript Payloads with Code Splitting {: .page-title }
+
+{% include "web/_shared/contributors/jeremywagner.html" %}
+{% include "web/_shared/contributors/addyosmani.html" %}
+
+**TL;DR:**
+
+- Modern apps often combine all of their JavaScript into a single, large bundle.
+When JavaScript is served this way, download and processing times can be
+significant on mobile devices and networks.
+- An alternative to large bundles is code-splitting, which is where JavaScript is
+split into smaller chunks. This enables sending the minimal code required to
+provide value upfront, improving page-load times. The rest can be loaded on
+demand.
+- Do you need code-splitting? Check Lighthouse's [JavaScript Bootup Time is Too
+High audit](/web/tools/lighthouse/audits/bootup) and [the code coverage
+panel](/web/updates/2017/04/devtools-release-notes#coverage) in DevTools to
+measure the impact of your app's scripts on performance and how many scripts are
+unused.
+- Code-splitting can be done in the following ways:
+    - Vendor splitting separates vendor code (e.g., React or lodash) away from your
```

Comment:
Can we bolden the start-of-line terms in this list for readability? (e.g. "Vendor splitting", "Entry point splitting" etc)